definitions/subannual/tag_time_input_types.yaml

```diff
@@ -0,0 +1,189 @@
+- Season Input:
+  - Winter:
+      duration: 90 / 365
+      description: period between January 1st (included) and March 20 (included) plus period between Dec. 21 (included)  and Dec 31 (included)
+
+  - Spring:
+      duration: 92 / 365
+      description: period between March 21 (included) and June 20 (included)
+
+  - Summer:
+      duration: 92 / 365
+      description: period between June 21 (included) and Sept. 20 (included)
+
+  - Autumn:
+      duration: 91 / 365
+      description: period between Sept 21 (included) and Dec. 20 (included)
+      
+# Hourly counter within one week 
+- Week Hour Index Input:
```

Comment:
@erikfilias what do you mean by label? 
What I need is to be able to describe representative weeks and average weeks; this means that instead of uploading data for 52 weeks (thus 52*168 rows as my data are hourly), I will only upload 4*168 rows (also because this is more representative and this is how we have presented eg the marginal costs in the report). So I need to express that a row corresponds to eg. the 15th hour of the average winter week. Which is not sth I can write in a format like 2050-01-12 12:00 ..... 
So if you have any idea how to better do that I will be very gratefull (thought of it for days but did not find anything better.....)